src/components/ContactsList.js

```diff
@@ -1,10 +1,26 @@
-import { useState } from "react"
-import { Link, useSearchParams } from "react-router-dom"
+import { useState } from "react";
+import { Link, useSearchParams } from "react-router-dom";
 
 function ContactsList(props) {
-  
   //"contacts" must be passed as prop to this component
-  const { contacts } = props
+  const { contacts, setContacts } = props;
+
+  function handleDelete(contact) {
+    // Delete the contact
+    fetch(`http://localhost:4000/contacts/${contact.id}`, {
+      method: "DELETE",
+    })
+      .then(function (response) {
+        return response.json();
+      })
+      .then(function () {
+        const filteredContacts = contacts.filter(
+          (contactInfo) => contactInfo.id !== contact.id
```

Comment:
nice method to deal with deleted and updating the database, then rendering again the contacts list

simple and kept it to only a few lines